be/src/vec/columns/predicate_column.h

```diff
@@ -91,13 +91,17 @@ class PredicateColumnType final : public COWHelper<IColumn, PredicateColumnType<
     void insert_string_to_res_column(const uint16_t* sel, size_t sel_size,
                                      vectorized::ColumnString* res_ptr) {
         StringRef refs[sel_size];
+        size_t length = 0;
         for (size_t i = 0; i < sel_size; i++) {
             uint16_t n = sel[i];
             auto& sv = reinterpret_cast<StringValue&>(data[n]);
             refs[i].data = sv.ptr;
             refs[i].size = sv.len;
+            length += sv.len;
         }
-        res_ptr->insert_many_continuous_strings(refs, sel_size);
```

Comment:
only here to call `insert_many_continuous_strings` after delete the code, we should del the funciton